packages/components/src/custom/Modal/modal.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, ReactNode } from 'react';
+import {
+  StyledDialog,
+  StyledDialogActions,
+  StyledDialogContent,
+  StyledDialogTitle
+} from '../Dialog';
+
+interface MesheryModalProps {
+  open: boolean;
+  content: ReactNode;
+  title: string;
+  actions?: ReactNode;
+}
+
+const MesheryModal: FC<MesheryModalProps> = ({ open, content, title, actions }) => {
+  return (
+    <StyledDialog fullWidth maxWidth="md" open={open}>
+      <StyledDialogTitle>{title}</StyledDialogTitle>
```

Comment:
@coder12git below image shows how our custom modal looks like currently, can you make sure the things I mentioned with red square in this image are added in this modal as these are going to be reusable

For reference, you can look at [Modal.js](https://github.com/meshery/meshery/blob/master/ui/components/Modal.js) file in meshery

![Screenshot 2023-11-01 at 7 20 46 AM](https://github.com/layer5io/sistent/assets/71271069/0e7f9e37-453e-4e29-b159-032066d7ac90)

It might be possible that gradient is not updated in meshery modal, so here is style for header background
```
  boxShadow: "inset 0px -1px 3px 0px rgba(0,0,0,0.2)",
  background: "linear-gradient(90deg, rgba(57,102,121,1) 0%, rgba(84,129,148,1) 100%)",
  filter:
    "progid:DXImageTransform.Microsoft.gradient(startColorstr='#396679',endColorstr='#548194',GradientType=1)",
```

for footer

```
  backgroundColor: "#396679",
  boxShadow: "inset 0px 3px 5px 0px rgba(0,0,0,0.25)"
```

`Left icon` and `helperIcon` are going to be conditionally rendered on the basis of props given to this modal, see this code for reference https://github.com/meshery/meshery/blob/2e31041178f3d666e8bad6f9f89bdd4ec21ab609/ui/components/Modal.js#L217

Reply:
Sure, I will do the changes!